query/src/configs/config_meta.rs

```diff
@@ -37,6 +46,10 @@ pub struct MetaConfig {
     #[serde(alias = "meta_address")]
     pub address: String,
 
+    #[clap(long, env = META_ENDPOINTS, default_value = "", help = "MetaStore peers endpoints")]
```

Comment:
- `env = META_ENDPOINTS` is not needed anymore.
- And it perfer to use `default_value_t` instead.
- To make `clap` works correctly, we need `long="meta-endpoints"`